GraphQL type for ´Zone` has wrong representation for `nameservers`

**Versions**
NetBox Version: 4.0.6
NetBox DNS Version: 1.0.3
Python Version: 3.11.5

**Describe the bug**
In the GraphQL `NetBoxDNSZoneType`, the `nameservers` field is not specified as a `List`.

**To Reproduce**
Upgrade to NetBox 4.0.6 and run the test suite.

**Expected result**
All tests succeed.

**Actual result**
```
======================================================================
FAIL: test_graphql_list_objects (netbox_dns.tests.zone.test_api.ZoneAPITestCase.test_graphql_list_objects)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/opt/netbox/lib64/python3.11/site-packages/django/test/utils.py", line 443, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/netbox/lib64/python3.11/site-packages/django/test/utils.py", line 443, in inner
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/netbox/netbox/utilities/testing/api.py", line 581, in test_graphql_list_objects
    self.assertNotIn('errors', data)
AssertionError: 'errors' unexpectedly found in {'data': {'netbox_dns_zone_list': [{'changelog': [], 'display': 'zone1.example.com', 'class_type': 'Zone', 'tags': [], 'journal_entries': [], 'custom_fields': {}, 'name': 'zone1.example.com', 'status': 'active', 'active': True, 'view': {'id': '1'}, 'nameservers': None, 'default_ttl': 86400, 'soa_ttl': 86400, 'soa_mname': {'id': '556'}, 'soa_rname': 'hostmaster.example.com', 'soa_serial': 1, 'soa_refresh': 43200, 'soa_retry': 7200, 'soa_expire': 2419200, 'soa_minimum': 3600, 'soa_serial_auto': False, 'description': '', 'arpa_network': None, 'tenant': None, 'registrar': None, 'registry_domain_id': None, 'registrant': None, 'admin_c': None, 'tech_c': None, 'billing_c': None, 'rfc2317_prefix': None, 'rfc2317_parent_managed': 'false', 'rfc2317_parent_zone': None, 'id': '1068', 'created': '2024-06-25T08:46:37.885092+00:00', 'last_updated': '2024-06-25T08:46:37.885106+00:00', 'custom_field_data': {}}, {'changelog': [], 'display': 'zone2.example.com', 'class_type': 'Zone', 'tags': [], 'journal_entries': [], 'custom_fields': {}, 'name': 'zone2.example.com', 'status': 'active', 'active': True, 'view': {'id': '1'}, 'nameservers': None, 'default_ttl': 86400, 'soa_ttl': 86400, 'soa_mname': {'id': '556'}, 'soa_rname': 'hostmaster.example.com', 'soa_serial': 1, 'soa_refresh': 43200, 'soa_retry': 7200, 'soa_expire': 2419200, 'soa_minimum': 3600, 'soa_serial_auto': False, 'description': '', 'arpa_network': None, 'tenant': None, 'registrar': None, 'registry_domain_id': None, 'registrant': None, 'admin_c': None, 'tech_c': None, 'billing_c': None, 'rfc2317_prefix': None, 'rfc2317_parent_managed': 'false', 'rfc2317_parent_zone': None, 'id': '1069', 'created': '2024-06-25T08:46:37.941158+00:00', 'last_updated': '2024-06-25T08:46:37.941174+00:00', 'custom_field_data': {}}, {'changelog': [], 'display': '[view1] zone3.example.com', 'class_type': 'Zone', 'tags': [], 'journal_entries': [], 'custom_fields': {}, 'name': 'zone3.example.com', 'status': 'active', 'active': True, 'view': {'id': '49'}, 'nameservers': None, 'default_ttl': 86400, 'soa_ttl': 86400, 'soa_mname': {'id': '556'}, 'soa_rname': 'hostmaster.example.com', 'soa_serial': 1, 'soa_refresh': 43200, 'soa_retry': 7200, 'soa_expire': 2419200, 'soa_minimum': 3600, 'soa_serial_auto': False, 'description': '', 'arpa_network': None, 'tenant': None, 'registrar': None, 'registry_domain_id': None, 'registrant': None, 'admin_c': None, 'tech_c': None, 'billing_c': None, 'rfc2317_prefix': None, 'rfc2317_parent_managed': 'false', 'rfc2317_parent_zone': None, 'id': '1070', 'created': '2024-06-25T08:46:37.997780+00:00', 'last_updated': '2024-06-25T08:46:37.997792+00:00', 'custom_field_data': {}}, {'changelog': [], 'display': '[view2] zone4.example.com', 'class_type': 'Zone', 'tags': [], 'journal_entries': [], 'custom_fields': {}, 'name': 'zone4.example.com', 'status': 'active', 'active': True, 'view': {'id': '50'}, 'nameservers': None, 'default_ttl': 86400, 'soa_ttl': 86400, 'soa_mname': {'id': '556'}, 'soa_rname': 'hostmaster.example.com', 'soa_serial': 1, 'soa_refresh': 43200, 'soa_retry': 7200, 'soa_expire': 2419200, 'soa_minimum': 3600, 'soa_serial_auto': False, 'description': '', 'arpa_network': None, 'tenant': None, 'registrar': None, 'registry_domain_id': None, 'registrant': None, 'admin_c': None, 'tech_c': None, 'billing_c': None, 'rfc2317_prefix': None, 'rfc2317_parent_managed': 'false', 'rfc2317_parent_zone': None, 'id': '1071', 'created': '2024-06-25T08:46:38.027731+00:00', 'last_updated': '2024-06-25T08:46:38.027740+00:00', 'custom_field_data': {}}, {'changelog': [], 'display': '[view3] zone5.example.com', 'class_type': 'Zone', 'tags': [], 'journal_entries': [], 'custom_fields': {}, 'name': 'zone5.example.com', 'status': 'active', 'active': True, 'view': {'id': '51'}, 'nameservers': None, 'default_ttl': 86400, 'soa_ttl': 86400, 'soa_mname': {'id': '556'}, 'soa_rname': 'hostmaster.example.com', 'soa_serial': 1, 'soa_refresh': 43200, 'soa_retry': 7200, 'soa_expire': 2419200, 'soa_minimum': 3600, 'soa_serial_auto': False, 'description': '', 'arpa_network': None, 'tenant': None, 'registrar': None, 'registry_domain_id': None, 'registrant': None, 'admin_c': None, 'tech_c': None, 'billing_c': None, 'rfc2317_prefix': None, 'rfc2317_parent_managed': 'false', 'rfc2317_parent_zone': None, 'id': '1072', 'created': '2024-06-25T08:46:38.051784+00:00', 'last_updated': '2024-06-25T08:46:38.051796+00:00', 'custom_field_data': {}}]}, 'errors': [{'message': "Expected value of type 'NetBoxDNSNameServerType' but got: [].", 'locations': [{'line': 13, 'column': 12}], 'path': ['netbox_dns_zone_list', 0, 'nameservers']}, {'message': "Expected value of type 'NetBoxDNSNameServerType' but got: [].", 'locations': [{'line': 13, 'column': 12}], 'path': ['netbox_dns_zone_list', 1, 'nameservers']}, {'message': "Expected value of type 'NetBoxDNSNameServerType' but got: [].", 'locations': [{'line': 13, 'column': 12}], 'path': ['netbox_dns_zone_list', 2, 'nameservers']}, {'message': "Expected value of type 'NetBoxDNSNameServerType' but got: [].", 'locations': [{'line': 13, 'column': 12}], 'path': ['netbox_dns_zone_list', 3, 'nameservers']}, {'message': "Expected value of type 'NetBoxDNSNameServerType' but got: [].", 'locations': [{'line': 13, 'column': 12}], 'path': ['netbox_dns_zone_list', 4, 'nameservers']}]}
```

--- a/netbox_dns/graphql/types.py
+++ b/netbox_dns/graphql/types.py
@@ -1,4 +1,4 @@
-from typing import Annotated
+from typing import Annotated, List
 
 import strawberry
 import strawberry_django
@@ -37,12 +37,11 @@ class NetBoxDNSZoneType(NetBoxObjectType):
     status: str
     active: bool
     view: Annotated["NetBoxDNSViewType", strawberry.lazy("netbox_dns.graphql.types")]
-    nameservers: (
+    nameservers: List[
         Annotated[
             "NetBoxDNSNameServerType", strawberry.lazy("netbox_dns.graphql.types")
         ]
-        | None
-    )
+    ]
     default_ttl: BigInt
     soa_ttl: BigInt
     soa_mname: Annotated[

